clang/test/Driver/mips-mti-linux.c

```diff
@@ -12,7 +12,7 @@
 // RUN:     --sysroot=%S/Inputs/mips_mti_linux/sysroot \
 // RUN:   | FileCheck --check-prefix=CHECK-BE-HF-32R2 %s
 //
-// CHECK-BE-HF-32R2: "{{[^"]*}}clang{{[^"]*}}" {{.*}} "-triple" "mips-mti-linux"
```

Comment:
Instead of changing the check, can you change the `RUN` lines to use `%clang -no-canonical-prefixes`? I would expect that to not realpath-ify the binary name.

Reply:
Thanks, I wasn't aware of the `-no-canonical-prefixes` option, it fixes the failures.